flang/lib/Lower/OpenMP.cpp

```diff
@@ -50,14 +50,13 @@ static void createPrivateVarSyms(Fortran::lower::AbstractConverter &converter,
 
     // Privatization for symbols which are pre-determined (like loop index
     // variables) happen separately, for everything else privatize here
-    if (!sym->test(Fortran::semantics::Symbol::Flag::OmpPreDetermined)) {
-      [[maybe_unused]] bool success =
+    // if (!sym->test(Fortran::semantics::Symbol::Flag::OmpPreDetermined)) {
```

Comment:
Left over code or code that should be added back ?

Reply:
Thanks. I should remove that. ;)